src/main/java/org/opentripplanner/graph_builder/GraphBuilderDataSources.java

```diff
@@ -106,7 +107,7 @@ public Iterable<DataSource> get(FileType type) {
     return inputData.get(type);
   }
 
-  public Iterable<ConfiguredDataSource<OsmExtractParameters>> getOsmConfiguredDatasource() {
+  public Collection<ConfiguredDataSource<OsmExtractParameters>> getOsmConfiguredDatasource() {
```

Comment:
I think the reason it is an `Iterable` and not `Collection` is that the oder matter - if not, at least for having a deterministic behaviour.

Reply:
I'm not sure I follow that logic: `Set` is also an `Iterable` and there the order is not guaranteed at all. 

To keep unrelated discussion down, I have reverted the change: 49aba50de084dacdc65d1252e37fa83ec7b81574

Reply:
The `Iterable` *forces* the caller to iterate over the result in a specific order decided by the callee. I do not remember if we relay on it, but I do not want to spend time on it either - this is the stuff you must check if you change the code. Of cause it is bad to rely on such conventions, but if there is no real gain in chaning the code - then don´t. It is realy easy to change the code, but it takes a lot of time to review it - because I do not know if there was a bussiness reason for doing so. When I stumble across these changes I MUST assume you changed it to achive a bussniss change, related to you PR.